scripts/deploy_wordpot.sh

```diff
@@ -55,4 +55,11 @@ redirect_stderr=true
 stopsignal=QUIT
 EOF
 
+# Quick check to see if on Ubuntu using systemd or init
+if [ -n `which service` ]; then
```

Comment:
You need to wrap these in quotes otherwise the 1st if statement will always get executed.

```
$ which asdasd
$ if [ -n `which asdasd` ]; then echo ha; fi
ha
$ if [ -n "`which asdasd`" ]; then echo ha; fi
```